src/lib/monitoring/config.ts

```diff
@@ -987,6 +992,38 @@ export class ConfigGenerator {
     };
   }
 
+  public addGraphqlApis({ graphqlApis }: AWSItem): void {
+    if (graphqlApis.length === 0) {
+      return;
+    }
+
+    const defaultConfig = {
+      Latency: {
+        alarm: {
+          critical: {
+            threshold: 5000,
+            evaluationPeriods: 1,
+          },
+        },
+        metric: {
+          statistic: 'Average',
+        },
+      },
+    };
+
+    this.config = {
+      ...this.config,
+      appsyncApis: graphqlApis.reduce((acc, c) => ({ ...acc, [c.apiId || '']: {} }), {} as AlarmMetricConfig),
+      custom: {
+        ...this.config.custom,
+        default: {
+          ...this.config.custom.default,
+          appsyncApi: defaultConfig,
```

Comment:
I would use uppercased `S` (`appSyncApi`) for consistency (and do the same for other occurences of `*appSync` too in the PR).

Reply:
Good suggestion. I will fix this